matplotlib-stubs/pyplot.pyi

```diff
@@ -113,4 +115,38 @@ def subplots_adjust(
 ) -> None: ...
 def close(fig: Union[Figure, Literal["all"]]) -> None: ...
 def clf() -> None: ...
+def plot(*args: Any, scalex: bool = ..., scaley: bool = ..., data: Optional[_Data] = ..., **kwargs: Any) -> Line2D:
```

Comment:
Please copy the definition of `plot()` from `axes.pyi`. And then add any additional parameters as needed.